server/command.go

```diff
@@ -248,8 +255,43 @@ func (p *Plugin) runPopCommand(args []string, extra *model.CommandArgs) (bool, e
 	return false, nil
 }
 
+func (p *Plugin) runSettingsCommand(args []string, extra *model.CommandArgs) (bool, error) {
+	if args[0] == "summary" {
```

Comment:
I think there should be a bounds check on 'args' array here

Reply:
Added length check on arguments